Collectors/DetailedCollector.py

```diff
@@ -144,13 +147,22 @@ def addRecord(sid, userID, fileClose, timestamp, addr):
     rec['read'] = fileClose.read
     rec['readv'] = fileClose.readv
     rec['write'] = fileClose.write
+    
```

Comment:
Trailing whitespace here and on line 160.

Reply:
Ugh, at my IDE